core/lib/generators/spree/dummy/dummy_generator.rb

```diff
@@ -30,6 +30,13 @@ def generate_test_dummy
       opts[:database] = 'sqlite3' if opts[:database].blank?
       opts[:force] = true
       opts[:skip_bundle] = true
+      opts[:skip_gemfile] = true
+      opts[:skip_test] = true
+      opts[:skip_git] = true
+      opts[:skip_keeps] = true
+      opts[:skip_spring] = true
+      opts[:skip_listen] = true
+      opts[:skip_puma] = true
```

Comment:
looking at [here](https://github.com/rails/rails/blob/5-1-stable/railties/lib/rails/generators/app_base.rb) maybe we can also add `skip_yarn` and `no_rc `?